controllers/flinkcluster_converter.go

```diff
@@ -624,6 +633,46 @@ func shouldCleanup(
 	return false
 }
 
+func calFlinkHeapSize(cluster *v1alpha1.FlinkCluster) map[string]string {
```

Comment:
Could you add a unit test for it?

Reply:
Added it in flinkcluster_converter_test.go